dev-server/serve-dev.js

```diff
@@ -175,7 +175,7 @@ export function serveDev(port, config) {
   });
 
   // Serve UI component playground
-  app.get('/ui-playground/:path?', (req, res) => {
+  app.get('/ui-playground{/:path}', (req, res) => {
```

Comment:
Express uses `path-to-regexp` to match URLs. The version used by express 5 has some breaking changes, including the fact that optional segments are now enclosed in curly braces, instead of adding a question mark at the end.

See https://github.com/pillarjs/path-to-regexp?tab=readme-ov-file#unexpected--or-